README.md

```diff
@@ -315,3 +316,59 @@ Comp 40 gives you the tools to explore a large number of other topics:
 * Mark Sheldon gets very excited about some of the topics in Comp 40. Stop by his office to chat with him!
 * Norman Ramsey designed the class
 * Noah Mendelsohn is the [chairman of the internet](http://www.arcanedomain.com/computerwork.html) and is in general incredibly smart; he teaches COMP 40 on alternating semesters with Prof. Sheldon. Talk to him too!
+
+## COMP 15
+
+### Topics Covered:
+* Your first guide to getting a software engineering job (interview questions!)
+* Knowing the difference between the client side code (using functions without knowing its implementation details) and the implementation side code (the code that runs your functions as promised in your documentation)
+* Introduction to C++ programs that utilize complex data structures
+* Utilizing Object Oriented Programming (OOP) concepts to write codes that are short and easy to understand
+* Using abstraction/modularity to break down complex problems
+* Allocating and managing dynamic memory and visualizing what happens under the hood (aliasing, deep vs shallow copies)
+* Evaluating different data structures (sequences vs linked list, stacks vs queues) according to their time and space complexity
+* Manipulating previous data structures (sequences, linked lists) to make more complex data structures (trees, heaps and more!)
+* Writing code that must handle MANY test cases and handling exceptions
+* Understanding Big O (asymptotic analysis) and why certain choices are better (first step towards becoming an ‘engineer’)
+* Reading long documentations (~8 pages PDF) for homework assignments
+* Writing technical comments and documents (README) that make your code easier to follow
+* RECURSION, recursion, ………, [recursion](https://www.google.com/search?q=aesthetics+of+being+short&rlz=1C5CHFA_enUS710US710&oq=aesthetics+of+being+short&aqs=chrome..69i57.3764j0j4&sourceid=chrome&ie=UTF-8#safe=off&q=recursion) has many merits beyond its aesthetics, which is one of the reasons why many programmers consider them important. It's a unique way of thinking that you should really familiarize yourself with
+* Debugging at a new level and mastering how segmentation fault works
```

Comment:
Does COMP 15 really emphasize mastering how segmentation faults work? Perhaps instead: "learning what can cause a segmentation faults in more detail"